packages/expo-gl/EXGL.podspec

```diff
@@ -1,6 +1,6 @@
 require 'json'
 
-package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
```

Comment:
any reason for putting the podspec in the package root folder and disable rn-cli autolinking? since in most of expo modules, we put podspec inside *ios/* folder.

Reply:
CocoaPods doesn't support going up (`../`) in the source files spec property 😢 We do the same in `expo-modules-core`

Reply:
oh right! thanks for the reminder 👍